controllers/designateunbound_controller.go

```diff
@@ -125,6 +125,7 @@ func (r *UnboundReconciler) Reconcile(ctx context.Context, req ctrl.Request) (re
 	)
 
 	instance.Status.Conditions.Init(&cl)
+	instance.Status.ObservedGeneration = instance.Generation
```

Comment:
you are missed the generation check for the deployment created by this controller. https://github.com/openstack-k8s-operators/designate-operator/blob/29d27af2376793a54cde1cd9abbb867d71ac1290/controllers/designateunbound_controller.go#L379-L382

Reply:
good catch, thank you!

Reply:
Done